hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/SpanTypeAttributeEnricher.java

```diff
@@ -89,7 +95,16 @@ public void enrichEvent(StructuredTrace trace, Event event) {
 
     // Figure out if event is entry or exit based on other span attributes
     Boolean isEntry = null;
-    if (attributeMap.containsKey(SPAN_KIND_KEY)) {
+    if (attributeMap.containsKey(OTEL_SPAN_KIND)) {
```

Comment:
Pl. wrap this in `spansemanticConventions.util` too. This will make things consistent.

Reply:
I will take up refactoring this, once otel support is complete for all repos.